### PR TITLE
Set up forge docs

### PR DESCRIPTION
**Pull request by haydenshively**
See deployment at https://aloelabs.github.io/aloe-ii/

Once merged, we can remove this branch from the allowed deployers list.

Currently intended to supplement our documentation on Gitbooks (the contract section in particular) not replace it.

### CODE REVIEW

File: core/src/Ledger.sol [hunk 1]
```diff
@@ -259,15 +259,10 @@ contract Ledger {
     /**
      * @notice Returns a conservative estimate of the maximum amount of `asset()` that can be deposited into the
      * Vault for `receiver`, through a deposit call.
-     * @return The maximum amount of `asset()` that can be deposited
-     *
-     * @dev Should return the *precise* maximum. In this case that'd be on the order of 2**112 with weird constraints
+     * @dev Should return the *precise* maximum. In this case that'd be on the order of 2^112 with weird constraints
```
Comment (aneesh1) [inline core/src/Ledger.sol]: word choice: `weird` constraints?

File: core/src/Ledger.sol [hunk 2]
```diff
@@ -276,15 +271,10 @@ contract Ledger {
     /**
      * @notice Returns a conservative estimate of the maximum number of Vault shares that can be minted for `receiver`,
      * through a mint call.
-     * @return The maximum number of Vault shares that can be minted
-     *
-     * @dev Should return the *precise* maximum. In this case that'd be on the order of 2**112 with weird constraints
+     * @dev Should return the *precise* maximum. In this case that'd be on the order of 2^112 with weird constraints
```
Comment (aneesh1) [inline core/src/Ledger.sol]: wc (again): weird constraints?

File: core/src/libraries/constants/Constants.sol [hunk 3]
```diff
@@ -10,15 +10,17 @@ uint256 constant BORROWS_SCALER = type(uint72).max * ONE;
 
 /// @dev The maximum percentage yield per second, scaled up by 1e12. The current value is equivalent to
 /// (1 + 706354 / 1e12) ^ (24 * 60 * 60) - 1 ⇒ +6.3% per day or +53% per week. If the rate is consistently at this
```
Comment (aneesh1) [inline core/src/libraries/constants/Constants.sol]: Can we put this in TeX? 🥺

**Review by aneesh1 — APPROVED** [review]
LGTM. Couple of nits.

File: core/src/libraries/constants/Constants.sol (hunk 3 above)
Comment (haydenshively) [reply core/src/libraries/constants/Constants.sol]: I changed it to Python-style in-line code. Hopefully that's ok!

**Pull request merged**